test/services/mapping/governmentagencygoodsitem/StatisticalValueAmountBuilderSpec.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.mapping.governmentagencygoodsitem
+
+class StatisticalValueAmountBuilderSpec {}
```

Comment:
Presume this needs implementing

Reply:
TODO yeah! :) noticed this wasn;t being tested so wrote this then forgot to implement when I entered rebase hell. Will implement as part of next piece of work.